### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Alex Berberat and Lisa Gorgerat

### DISCUSSION THREAD

**Discussion by AlexB-HEIG** [heig-vd-dai-course/heig-vd-dai-course#358] [Show and tell]
A CLI tool to count and sort in different ways all the words from a text file.

Sorting example : 
- Alphabetical and reverse order
- Most to least used
- Apparition order

**Comment by ludelafo** [comment]
Looks good to us, thank you!

**Comment by LisaGorgerat** [comment]
link to our repository: https://github.com/AlexB-HEIG/DAI-Practical-work-1
hash of the last commit: f3487ce05ff33683d4f52993684f459337d92ef1
